packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -17,46 +17,49 @@
   --border-color: transparent;
 }
 
-.tag:hover {
+.root:hover {
   --bg-color: var(--color-surface--0);
   --border-color: var(--color-border-3);
 }
 
 /* Different Tag sizes */
 
 .small {
-  gap: var(--size-tiny);
-  padding: var(--size-tiny) var(--text-xxs);
-  font: normal 700 var(--text-xxs) Inter, sans-serif;
-  line-height: var(--size-tiny);
+  gap: var(--gap-tiny);
+  padding: var(--padding-tiny) var(--padding-xxs);
+  font: var(--font-inter);
+  font-weight: var(--font-weight-bold);
+  font-size: var(--text-xxs);
 }
 
 .medium {
-  gap: var(--size-tiny);
-  padding: var(--size-xxs) var(--text-xs);
-  font: normal 700 var(--text-xs) Inter, sans-serif;
-  line-height: var(--text-m);
+  gap: var(--gap-tiny);
+  padding: var(--padding-xxs) var(--text-xs);
+  font: var(--font-inter);
+  font-weight: var(--font-weight-bold);
+  font-size: var(--text-xs);
 }
 
 .large {
```

Comment:
I don't think we have this size designed or used anywhere? If we did, the padding proportions are off from the designs.

Reply:
Removed the large one